src/System.IO.Ports/src/System/IO/Ports/SerialStream.Win32.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+
+namespace System.IO.Ports
+{
+    internal sealed partial class SerialStream
+    {
+        public SafeFileHandle OpenPort(uint portNumber)
+        {
+            return Interop.Kernel32.CreateFileDefaultSecurity(
+                @"\\?\COM" + portNumber,
```

Comment:
Thanks for spotting it. I will change it here too but the important thing is in master since we build UWP from this branch.